_sass/api_documentation.scss

```diff
@@ -0,0 +1,100 @@
+/**
+Custom CSS for API Method Documentation pages
+
+See api/rest/rest_api_template.md for an example of how these are used.
+**/
+
+//custom REQUIRED label for parameters
+required-tag {
```

Comment:
Nit: can try to use some formatter to format this file. For example, can make the indent 2 spaces to match other files and align curly bracket.